etc/config/v.amazon.properties

```diff
@@ -0,0 +1,18 @@
+# Disabled: v v04
```

Comment:
this disabled comment should not be needed

Reply:
Without this comment, the pre-commit hook fails:
![image](https://github.com/compiler-explorer/compiler-explorer/assets/58859306/91beec2c-abab-4384-9ea2-f18ee61d5021)

Reply:
on line 6 you have `defaultCompiler=v`, this should be `defaultCompiler=v04` and on line 12 it should be `group.v.compilers=v04` - that should fix these errors

Reply:
awesome, fixed!